apps/android/firefox.py

```diff
@@ -5,18 +5,27 @@
 from mozprofile import create_profile
 
 
-class AndroidFirefox(object):
+class AbstractAndroidFirefox(object):
     def __init__(self, proxy, certutil):
         self.proxy = proxy
         self.certutil = certutil
+        self.app_args = [
+            "--marionette",
+            "--es",
+            "env0",
+            "LOG_VERBOSE=1",
+            "--es",
+            "env1",
+            "R_LOG_LEVEL=6",
+        ]
+        self.profile = None
 
-    def start(self, url="about:blank"):
-        # create profile
-        profile = create_profile("firefox")
-        print("Created profile: {}".format(profile.profile))
+    def set_profile(self):
+        self.profile = create_profile("firefox")
+        print("Created profile: {}".format(self.profile.profile))
```

Comment:
please use the same pattern through the hole file
ex: print statement before or after action is made

Reply:
I put here after the action because it references the object created by the action.